packages/grid/_modules_/grid/components/column-header-item.tsx

```diff
@@ -44,7 +42,21 @@ export const ColumnHeaderItem = React.memo(
       });
     }
 
-    const handleResize = onResizeColumn ? () => onResizeColumn(column) : undefined;
+    const handleResize =
+      onResizeColumn && !disableColumnReorder ? () => onResizeColumn(column) : undefined;
+    const handleDragStart =
+      onColumnDragStart && !disableColumnReorder
+        ? (event) => onColumnDragStart(column, event.currentTarget)
+        : undefined;
```

Comment:
A ternary operator can't finish with undefined as it's the default value so it's ugly.
Can be refactored to 
`
const handleDragStart = !disableColumnReorder
      onColumnDragStart && !disableColumnReorder
    &&  (event) => onColumnDragStart(column, event.currentTarget);
`

Reply:
Done